azurecosmos/README.md

```diff
@@ -0,0 +1,120 @@
+<!--
+Copyright (c) 2010 Yahoo! Inc., 2012 - 2018 YCSB contributors.
```

Comment:
Copyright statement should just say "2018 YCSB contributors" (especially for brand new files in this PR).

Reply:
Similar comments for all copyright statements below.

Reply:
fixed.  thanks for catching these.